ctutil/sctscan/sctscan.go

```diff
@@ -98,14 +98,15 @@ func main() {
 	}
 	s := scanner.NewScanner(logClient, scanOpts)
 
-	s.Scan(ctx,
+	if s.Scan(ctx,
 		func(entry *ct.RawLogEntry) {
 			checkCertWithEmbeddedSCT(ctx, logsByHash, *inclusion, entry)
 		},
 		func(entry *ct.RawLogEntry) {
 			glog.Errorf("Internal error: found pre-cert! %+v", entry)
-		})
-
+		}) != nil {
+		glog.Exitf("Scan failed: %v", err)
```

Comment:
You don't actually assign the error to `err`, so this will log the wrong thing (nil).

Reply:
Oops, fixed.